TYPO3.TYPO3CR/Classes/TYPO3/TYPO3CR/Domain/Service/ImportExport/NodeImportService.php

```diff
@@ -512,6 +512,9 @@ protected function persistEntities($propertyValue) {
 	 */
 	protected function parseEndElement(\XMLReader $reader) {
 		switch ($reader->name) {
+			case 'hiddenBeforeDateTime':
+			case 'hiddenAfterDateTime':
+			case 'contentObjectProxy':
```

Comment:
why contentObjectProxy? it's not present in parseElement, so should throw an exception if present.. I would remove it and create a separate change